### ISSUE TITLE
Some minor things that you probably know about.

### ISSUE THREAD

**Issue by Ternega** [chummer5a/chummer5a#454]
Just some glitches I found while using your brilliant work.

* Advanced Lifestyle: Corporate Owned is illegal even if you have either of proper SINs 
* Magic spells: When adding spells selecting a category with double click to open instead of hitting "+" causes error, it looks like it tries to pull category like it would a spell and understandably comes up empty
* Priority Chargen: You can purchase skill with Karma and then spec it with skill points, this is illegal.
* Mortimer of London: This is minor but Coats demand an item to be paired with, whereas in book it seems optional
* Fancy blocks CS: If you save it as HTML some tables break due to wrong colspan values. In stat box, CM and Magic lines, Ammo lines, Spell Categories. Also it would be neat to have a button that can hide boxes you don't want like the base CS.

Items you might want to add:
* Concealed quick draw holster RG 54 275Y prorbably at [5] capacity unless you can find the actual number for it.
* Magic Lodge
* Sound link RF299 25Y - probably optional rule that no one will use

If you are interested, I will continue adding stuff should I stumble on it.

**Comment by joha4270** [comment]
>Priority Chargen: You can purchase skill with Karma and then spec it with skill points, this is illegal.

Do you have a source for this? I have see no souch thing.

>Fancy blocks CS 

This is made by a 3rd party, so no promises here.

>Magic Lodge

Magical Lodge Materials exists already

Rest of it is new, will look at it, but probably won't get it fixed in time for 176

**Comment by Ternega** [comment]
That was quick.

>Do you have a source for this? I have see no souch thing.
Could be me misinterpreting RAW but here is why I see it as illegal: Skills purchased with karma at Step 7, Skillpoints are spent at Step 5. The book makes clear distinction between the two for example: 
>The first number in the skills column is the number of 
skill points a character has to spend on individual skills. 
These skill points are generally used to purchase Active 
skills, though they can be used for Knowledge and Lan-
guage skills too (see below). If you don’t get exactly the 
skill ratings you want in this step, remember that skills 
may also be raised with Karma at the end of character 
creation.
or
> In addition, skill groups cannot be 
broken up in this step, so individual skill points cannot 
raise the ratings of skills purchased as a group. (Note 
that skill groups can be broken up in Step Seven: Spend 
Your Left Over Karma (p. 98). 
 > Magical Lodge Materials exists already
Materials are not ready for use when purchased. Magical lodge is a distinct item It takes (Force) days to set one up, again this is minor just wanted to let you know.

**Comment by joha4270** [comment]
Well, personally i think you answer it yourself with your source. 

First in step 5 you raise a skill to (whatever). Then in step 7, you spend some karma to add a specialization afterwards, same effect as if saving 7 karma in chargen, then leveling it up afterwards. 
Well, without the limit of leaving chargen with limited karma but whatever...

>That was quick.

Github email alerts and push notifications on emails :smile:

**Comment by Ternega** [comment]
Just to make it clear I meant purchasing Ranks with karma and THEN purchasing spec with points rather than karma, since this seems to be RAI I'll stop arguing the point but I would also like to add this quote (CRB 98) as an argument before I do:

>Any  remaining 
Karma  can  now  be  invested 
in  smoothing  out  any  rough 
edges, picking up or improv-
ing  skills,  buying  additional 
spells,  acquiring  bound  spir-
its,  bonding  foci  to  be  used 
at the start of the game, pur-
chasing  contacts,  etc.

**Comment by joha4270** [comment]
Wait you mean buying Automatics 1 for karma, then adding a spec in Assault
Rifles with karma?

Yeah, that might be illegal.

Seems like i cannot read

**Comment by Ternega** [comment]
That's exactly what I mean't, yes.

**Comment by chummer5a** [comment]
Easiest way to handle this would be to put in a check at the validatecharacter step. If priority && skillvalue == 0 && skillkarmavalue != 0 && chkspec != purchased with karma, throw an error.

**Comment by Ternega** [comment]
A more problematic bug that I stumbled on - you cant regroup skill groups containing skills bought with skill points. I'm not sure as to exact nature of the bug so the description above is just how I stumbled upon it.
EX: Create a character with 6 Computers 6 Hardware 6 Software at chargen, **not grouped**. Mark as created. Can't group those skills, but if I add to him some 50 karma and purchase 1 rank in gymnastics, running and swimming, again not grouped, than save the file and reopen it I can purchase more ranks as a group rather than individual skills.

Also would be curious why this is considered an optional rule.

**Comment by joha4270** [comment]
Skill(groups) in general is broken. One of those days i'll rewrite it all,
until then i'm afraid of touching that code.

**Comment by joha4270** [comment]
And while i'm not sure why  it is an optional rule, it might be a holdover
from 4e

**Comment by Ternega** [comment]
Okay no issue, it's not like working around it is hard, just thought it was worth mentioning.

**Comment by Kyoto-Kid** [comment]
["Skill(groups) in general is broken."]

I wonder if that is what may be causing a skill to revert back to it's original rating when a skill group is broken with Karma.  For example after my character entered into career mode, I broke the stealth skill group when I added a rating level with Karma (getting the message that I was about to break the skill group).. After saving closing and later reopening the generator later, I noticed that rating for her Sneaking skill was back down to 2 even though the Karma for the increase to 3 was deducted.  

The only way I could fix it was to adjust the skill by +1 in the Improvements tab.
